play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukServiceNavigationSpec.scala

```diff
@@ -70,7 +70,7 @@ class GovukServiceNavigationSpec
       output.first().attr("href") shouldBe "#"
     }
 
-    "render ServiceNavigation with slotted content" in {
+    "render ServiceNavigation with slotted content out of the nav element" in {
```

Comment:
what about renaming this something like

"render the nav element even if there are no nav items when either of the navigation slots are provided"

Reply:
This test ensures we do **not** render the nav element, as neither the navigation nor a navigation Slot is added. Should I add a third test for when we have a nav element without it being slotted content?